src/bf/bloom_filter/basic.cc

```diff
@@ -96,4 +96,9 @@ void basic_bloom_filter::swap(basic_bloom_filter& other)
   swap(bits_, other.bits_);
 }
 
+bitvector const& basic_bloom_filter::storage() const
+{
+	return bits_;
```

Comment:
Looks great, thanks. One tiny nitpick: 2 spaces indentation please. Just push to the same branch again and I'll merge it.